libs/components/popovers/src/lib/modules/popover/popover-content.component.html

```diff
@@ -21,23 +21,17 @@
     }"
   >
     @if (popoverTitle) {
-      <header
+      <div
         class="sky-popover-header"
         [skyThemeClass]="{
           'sky-padding-even-default': 'default',
           'sky-padding-even-lg sky-margin-stacked-lg': 'modern'
         }"
       >
-        <h1
-          class="sky-popover-title"
-          [skyThemeClass]="{
-            'sky-font-heading-4': 'default',
-            'sky-font-emphasized': 'modern'
-          }"
-        >
+        <h4 class="sky-popover-title sky-font-heading-4">
           {{ popoverTitle }}
-        </h1>
-      </header>
+        </h4>
```

Comment:
Talked to Design about this change. Matt wanted this to be an `h4` and to have the same styles for both default and modern themes.